Assets/HoloToolkit-Examples/UX/Scripts/DialogExampleLaunchButton.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using HoloToolkit.Unity.Buttons;
+using HoloToolkit.UX.Dialog;
+using System.Collections;
+using UnityEngine;
+
+namespace HoloToolkit.Examples.UX
+{
+    /// <summary>
+    /// Demonstrates how to launch Dialog UI with different number of buttons
+    /// </summary>
+    public class DialogExampleLaunchButton : MonoBehaviour
+    {
+        [SerializeField]
+        private Dialog dialogPrefab;
+
+        [SerializeField]
+        private bool isDialogLaunched;
+
+        [SerializeField]
+        private GameObject resultText;
```

Comment:
no tooltip?

Reply:
this is in an example script... I am happy to file an issue to add tooltips / doc comments in the june release, if that seems reasonable. otherwise, we can ask @cre8ivepark to add them.